DatadogInternal/Sources/Utils/ReflectionMirror.swift

```diff
@@ -70,25 +70,25 @@ internal struct ReflectionMirror {
         }
     }
 
-    let subject: Any
+    internal let subject: Any
 
     /// The static type of the subject being reflected.
     ///
     /// This type may differ from the subject's dynamic type when this mirror
     /// is the `superclassMirror` of another mirror.
-    let subjectType: Any.Type
+    internal let subjectType: Any.Type
 
     /// A suggested display style for the reflected subject.
     let displayStyle: DisplayStyle
 
     /// A collection of `Child` elements describing the structure of the
     /// reflected subject.
-    let children: Children
+    internal let children: Children
 
     /// A mirror of the subject's superclass, if one exists.
-    var superclassMirror: ReflectionMirror? { _superclassMirror.lazy }
+    internal var superclassMirror: ReflectionMirror? { _superclassMirror.lazy }
 
-    var keyPaths: [String: Int]? { _keyPaths.lazy }
+    internal var keyPaths: [String: Int]? { _keyPaths.lazy }
```

Comment:
These can be `public` as well 👍

Reply:
They are similar to the standard `Mirror` interface and can be useful during reflection.